### PR TITLE
study view - context menu for actions, instead of icons

### PR DESCRIPTION
**Pull request by adamabeshouse**
image

image

### CODE REVIEW

**Comment by adamabeshouse** [comment]
@zhx828 @jjgao can you guys check out the heroku deployment and tell me if it looks good to you?

File: src/pages/studyView/chartHeader/ChartHeader.tsx [hunk 1]
```diff
@@ -49,10 +50,8 @@ export interface ChartControls {
 
 @observer
 export class ChartHeader extends React.Component<IChartHeaderProps, {}> {
-    constructor(props: IChartHeaderProps) {
-        super(props);
-    }
 
+    @observable menuOpen = false;
     @observable downloadMenuActive = false;
```
Comment (zhx828) [inline src/pages/studyView/chartHeader/ChartHeader.tsx]: We can get rid of this as well.

**Comment by jjgao** [comment]
Thanks, @adamabeshouse! I am wondering if a three-dot icon is enough?

**Comment by adamabeshouse** [comment]
@jjgao  like this?

image

**Pull request merged**